### PR TITLE
CDS-57073 Service Dashboard revamp - Phase 2

### PR DESCRIPTION
**Pull request by michaelcretzman**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ x] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* https://harness.atlassian.net/browse/CDS-57073

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/continuous-delivery/monitor-deployments/monitor-cd-deployments.md [hunk 1]
```diff
@@ -10,36 +10,131 @@ helpdocs_is_published: true
...
+## Individual service dashboards
 
-![](../cd-dashboards/static/monitor-cd-deployments-20.png)
+:::note
+
+Currently, the following service dashboard functionality is behind the feature flag `CDC_SERVICE_DASHBOARD_REVAMP_NG`. Contact [Harness Support](mailto:support@harness.io) to enable the feature.
+
+:::
+
+Click a service in the **Total Services** table drills down to show more service details:
```
Comment (douglas-j-bothwell) [inline docs/continuous-delivery/monitor-deployments/monitor-cd-deployments.md:48]:
```suggestion
Click a service in the **Total Services** table to drill down and show more service details:
```

File: docs/continuous-delivery/monitor-deployments/monitor-cd-deployments.md [hunk 2]
```diff
@@ -10,36 +10,131 @@ helpdocs_is_published: true
...
+
+- Aborted
+- Aborted by deployment freeze
+- Expired
+- Waiting
+- Failed
+
+Selecting **See Open Tasks** displays executions that require your attention.
+
+The **See Open Tasks** button is only displayed if there are tasks over the last 3 days.
```
Comment (douglas-j-bothwell) [inline docs/continuous-delivery/monitor-deployments/monitor-cd-deployments.md:66]: You're calling it a button here but a banner before the list...are these two different UI elements?

File: docs/continuous-delivery/monitor-deployments/monitor-cd-deployments.md [hunk 3]
```diff
@@ -10,36 +10,131 @@ helpdocs_is_published: true
...
+<docimage path={require('./static/4e748162f05ea1d64ee5c64ca924fc41afe6c61fce81ccbbd7f1d32a64b40a90.png')} width="60%" height="60%" title="Click to view full size image" />  
+
+Each environment card shows the following:
+
+- **Latest artifact:** the latest artifact deployed by this service to this environment.
+- **Drift detection:** drift detection displays any differences between the deployments to environments in an environment group. For example, in this drift detection, two environments are using different artifacts with this service:
+
+  <docimage path={require('./static/2f326c7718023be7ad2226f2ea3adb8a8eb772b7407304d134bbb3ffc8e473ad.png')} width="60%" height="60%" title="Click to view full size image" />  
+
+  Drift detection is helpful to detect inconsistencies. For example, when you deploy a service to an environment group and there are failures in one of the environments or someone performs a different deployment to one or more of the group's environments.
```
Comment (douglas-j-bothwell) [inline docs/continuous-delivery/monitor-deployments/monitor-cd-deployments.md:85]:
```suggestion
  Drift detection is helpful to detect inconsistencies. For example, you deploy a service to an environment group and there are failures in one of the environments. Another example: someone performs a different deployment to one or more of the group's environments.
```

File: docs/continuous-delivery/monitor-deployments/monitor-cd-deployments.md [hunk 4]
```diff
@@ -73,7 +168,7 @@ The Deployments dashboard provides deployment frequency, status, and duration da
 
 ![](../cd-dashboards/static/monitor-cd-deployments-24.png)
 
-Clicking on data points or Service and Project names brings up data tables.
+Clicking on data points or service and project names brings up data tables.
```
Comment (douglas-j-bothwell) [inline docs/continuous-delivery/monitor-deployments/monitor-cd-deployments.md:171]:
```suggestion
Clicking on data points, service names,  or project names brings up data tables.
```

**Review by douglas-j-bothwell — DISMISSED** [review]

File: docs/continuous-delivery/monitor-deployments/static/d0bba45d5e8be50eeea30bcccbdffd4b0ed6d2900977782a2176257f53dc38ee.png
Comment (Sagar-Rajput7) [inline docs/continuous-delivery/monitor-deployments/static/d0bba45d5e8be50eeea30bcccbdffd4b0ed6d2900977782a2176257f53dc38ee.png:1]: NIT: Similar to the above.

File: docs/continuous-delivery/monitor-deployments/static/46f68f86975b85364f0103b07622c4c9db84dea76e0d3c8820166bc015d09b1c.png
Comment (Sagar-Rajput7) [inline docs/continuous-delivery/monitor-deployments/static/46f68f86975b85364f0103b07622c4c9db84dea76e0d3c8820166bc015d09b1c.png:1]: NIT: `Instance` on the left, is pointing to the group, here I think we should either point to the Instance Count column (the last column on the left table) or we can even remove it.

File: docs/continuous-delivery/monitor-deployments/static/b46ca573dd4b036c3ab5189e6edda2fd86207f6401e8337a52f365da443007a9.png
Comment (Sagar-Rajput7) [inline docs/continuous-delivery/monitor-deployments/static/b46ca573dd4b036c3ab5189e6edda2fd86207f6401e8337a52f365da443007a9.png:1]: Some additional info here - 
- Clicking on `env3-group` will take us to that particular env or env group.
- Similarly clicking on `1 Day ago` can take us to that particular execution.

File: docs/continuous-delivery/monitor-deployments/static/eebdfe6a596f69e0fd2dca49ce04dd5659094aab8d76ef38886fa905659ffac9.png
Comment (Sagar-Rajput7) [inline docs/continuous-delivery/monitor-deployments/static/eebdfe6a596f69e0fd2dca49ce04dd5659094aab8d76ef38886fa905659ffac9.png:1]: Not sure if we should add it or not - 
- clicking on any card will further filter the below execution list. ( service + env ) (same for artifact card )

**Review by Sagar-Rajput7 — DISMISSED** [review]
Looks good to me, thanks!

File: docs/continuous-delivery/monitor-deployments/monitor-cd-deployments.md [hunk 5]
```diff
@@ -10,36 +10,141 @@ helpdocs_is_published: true
...
+
+  <docimage path={require('./static/b46ca573dd4b036c3ab5189e6edda2fd86207f6401e8337a52f365da443007a9.png')} width="60%" height="60%" title="Click to view full size image" />  
+
+  Clicking on the environment group name will take you to that group's settings. Clicking on the execution date (in this example, `1 day ago`) will take you to the pipeline execution details.
+- **Deployments:** each deployment of the service or a selected artifact. If you filter using **My Deployments** you see only those deployments that your user account initiated. 
+
+### View in table
+
+The **View in Table** setting shows all of the instances of the service in all of the environments used with the service. 
+
```
Comment (sourabh-awashti) [inline docs/continuous-delivery/monitor-deployments/monitor-cd-deployments.md:125]: we can also add here as note that we show latest information of environment in table view
For example if after instance deployment a environment name got changed, then we will show the latest name, similiarly for other fields

**Pull request merged**